ios/CodePush/RCTConvert+CodePushInstallMode.m

```diff
@@ -1,5 +1,10 @@
 #import "CodePush.h"
+
+#if __has_include("RCTConvert.h")
```

Comment:
I don't think #if #else would protect you from duplicate imports. The #if only checks for existence - if it exists, it will try to import it. That's why my hypothesis is that the ordering might matter here - you should order by your preference.